cogs/utils/db.py

```diff
@@ -172,6 +172,21 @@ async def set_member(guildID, memberID, option, value):
     await conn.close()
 
 
+# Returns members who have birthday on a given date
+async def get_members_bday_equals_date(date):
```

Comment:
This is more of a style choice, but this name could probably be shortened to just `get_members_by_bday`.